index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>MBTI</title>
+    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="css/style.css" rel="stylesheet">
+</head>
+<body>
+  <div class="well center-block col-md-6" style="float: none">
+  <h1 class="text-center">MBTI: Which Game of Thrones character are you?</h1>
+    <section class="text-center">
+    To see which Game of Thrones character you mostly match up with, try this game, a totally inaccurate variant on the Myers-Briggs style personality test.
+    </section>
+    <h2 class="text-center">Explanation</h2>
+    <article class="text-center">This is a popular way of determining personality types, assigning one of two traits each in four different categories to your personality: <br>Introversion (preferring solitude and small groups of people) or Extroversion (preferring socializing)<br>Intuition (gathering information through the mental and abstract) or Sensing (gathering information through the concrete and physical)<br> Feeling (making decisions based on empathy and emotion) or Thinking (making decisions based on detached rationality)<br>Perception (showing the outside world your intuition or sensing side) or Judgment (showing the outside world your feeling or thinking side). <br>Thus, there are 16 personality types, such as ISTJ.
```

Comment:
understood. will do it later because i plan to rewrite this entire block of text.